Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "octocrab"
-version = "0.29.0"
+version = "0.29.1"
```

Comment:
Can you remove the version upgrade? That's automatically handled by `release-plz`.